docs/publish-a-certain-collection-daily/script.liquid

```diff
@@ -1,37 +1,73 @@
-{% if event.preview %}
-  {
-    "action": {
-      "type": "shopify",
-      "options": [
-        "update",
-        [
-          "product",
-          12345
-        ],
-        {
-          "published": true
+{% assign collection = shop.collections[options.collection_id__required] %}
+
+{% if collection == blank and event.preview == false %}
```

Comment:
Nuance: `event.preview` is never false ([docs](https://docs.usemechanic.com/article/373-the-event-object)). It's either true, or the `event` object just doesn't have a `preview` property at all. I think the best way to express the intent here is `event.preview != true`.